data/remnant/remnant outfits.txt

```diff
@@ -474,6 +474,18 @@ outfit "Emergency Ramscoop"
 	"ramscoop" 0.5
 	description "The Ember Waste is the sort of place where only a very foolish captain would travel without some sort of device for replenishing hyperspace fuel. While this risk is non-existent in Remnant ships due to their 'gills', which are effectively built-in ramscoops, some captains prefer to speed up the process by adding additional capacity. For the rare captain braving the area in non-Remnant ships, this ramscoop is not particularly powerful, but it allows a ship to explore without fear of getting stranded."
 
+outfit "NDR-114 Android"
+	category "Systems"
+	licenses
+		Remnant
+	cost 120000
+	thumbnail "outfit/android"
+	"required crew" -1
+	"bunks" -1
+	unplunderable 1
+	description "While these were once considered cutting edge outfits, their lack of autonomous capabilities and economic benefit soon led to their commercial failure and disappearance from the market several centuries ago."
+	description	"	Despite these obstacles, some captains preferred replacing one of their crew members with a robotic counterpart, and considered the reduced bunk capacity a fair trade. These captains found that androids possessed sufficient capabilities to perform most work, but lacked the decision making capability to captain a ship or take part in boarding actions."
+
```

Comment:
May be best to give the reintroduced androids a very small bit of energy consumption?

Nothing drastic, just enough to demonstrate they do cause a drain.

Reply:
I could, although I have generally been of the opinion that their energy use would be in the same range as that consumed by life support

Reply:
From my experience doing the same sort of minute energy drain for the Void Rifle; the value either ends up being too small to notice, or big enough to be problematic. It's really hard to get a good balance on continuous-drain outfits. Weapons, cloak, and engines the player can *choose* not to spend the energy if they're running low. This would be an "always present" drain that could prove quite dangerous, especially during ion storms.